Grbl_Esp32/src/Motors/Motor.cpp

```diff
@@ -55,13 +56,17 @@ namespace Motors {
     }
 
     void Motor::set_disable(bool disable) {}
-    void Motor::set_direction_pins(uint8_t onMask) {}
-    void Motor::step(uint8_t step_mask, uint8_t dir_mask) {}
+    void Motor::set_direction(bool dir) {}
+    void Motor::step() {}
```

Comment:
If `set_direction`, `step` and `unstep` should always be implemented, we should make these pure virtual calls. Remove implementation, change signature to e.g. `virtual void step() = 0;` in the header file.